### PR TITLE
Remove the parameter: ResourceQuota

### PR DESCRIPTION
**Pull request by Byronliu123**
when I run the Powershell which includes Resource Quota parameter, the warning message: the parameter resourcequota is obsolete, so please remove it in the article.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/New-SPOSite.md [hunk 1]
```diff
@@ -129,22 +129,6 @@ Accept pipeline input: False
...
-Parameter Sets: (All)
-Aliases:
-Applicable: SharePoint Online
-
-Required: False
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/New-SPOSite.md:132]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Byronliu123, kindly commit the suggested change to remove the extra blank line. Many thanks.

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — COMMENTED** [review]
Hi @Byronliu123, thank you for your contribution.

When a parameter is still in the code but deprecated, we don't remove it from the article. Instead, we detail in the description that the parameter was deprecated.

**Review by get-itips — CHANGES_REQUESTED** [review]
> Hi @Byronliu123, thank you for your contribution.
> 
> When a parameter is still in the code but deprecated, we don't remove it from the article. Instead, we detail in the description that the parameter was deprecated.

Agreed

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Byronliu123 Gentle reminder.

**Comment by yogkumgit** [comment]
@Byronliu123 Please implement the suggested feedback so that merge can be initiated. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please help.

**Comment by tiburd** [comment]
Closing this PR based on reviewers' comments.

**Pull request closed without merging**